### DISCUSSION TITLE
installation problems on Linux mint

### DISCUSSION THREAD

**Discussion by susannalinux** [sorayuki/obs-multi-rtmp#418] [Q&A]
Hello community, I tried to install this plugin obs-multi-rtmp 0.5.0.4  in OBS 30.1.1 Flatpack / on Linux mint.
I checked also a lot of yt vids but it's still not working. Looking for any help please.

**Comment by sorayuki** [comment]
Check the OBS log file to see if it tried to load the plugin (make sure the plugin located in the right place)
If it shows fail to load, maybe some .so file is absent or in other names.
Maybe run OBS with debugger tools may help find what .so cause the problem.

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

________________________________
发件人: susannalinux ***@***.***>
发送时间: Wednesday, March 27, 2024 6:58:32 PM
收件人: sorayuki/obs-multi-rtmp ***@***.***>
抄送: Subscribed ***@***.***>
主题: [sorayuki/obs-multi-rtmp] installation problems on Linux mint (Discussion #418)


Hello community, I tried to install this plugin obs-multi-rtmp 0.5.0.4 in OBS 30.1.1 Flatpack / on Linux mint.
I checked also a lot of yt vids but it's still not working. Looking for any help please.

―
